app/main.py

```diff
@@ -37,4 +39,27 @@ def read_root(db: db_dependency):
 
 @app.post("/friend")
 def add_friend(friend: NewFriend, db: db_dependency):
-    return friend
+    db_friend = models.Friend(name=friend.name, location=friend.location, friends=friend.friends)
+    db.add(db_friend)
+    db.commit()
+    return 'Friend Added'
+
+@app.delete('/FRIENDS/{friend_id}')
```

Comment:
We should make these route handlers lowercase for consistency, e.g (`/friends/{friend_id}`)